src/controller/java/src/chip/devicecontroller/WriteAttributesCallbackJni.java

```diff
@@ -19,7 +19,7 @@
 
 /** JNI wrapper callback class for {@link WriteAttributesCallback}. */
 public class WriteAttributesCallbackJni {
-  private WriteAttributesCallback wrappedWriteAttributesCallback;
+  final private WriteAttributesCallback wrappedWriteAttributesCallback;
```

Comment:
nit: Comment for the `final private` field here and other files: the conventional field order is `private final`

https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers

Reply:
done, thanks